group-anagrams/HodaeSsi.py

```diff
@@ -0,0 +1,10 @@
+# 시간복잡도 : O(n*mlogm) (n은 strs의 길이, m은 strs의 원소의 길이)
+# 공간복잡도 : O(n)
+
+class Solution:
+	def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
+		anagrams = collections.defaultdict(list)
```

Comment:
```suggestion

from collections import defaultdict

class Solution:
	def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
		anagrams = defaultdict(list)
```

파이썬에서 보통은 이런 식으로 모듈을 import 하는데, 이러면 어떤 모듈을 사용하는 파일인지 알기 쉬워져서 가독성이 더 좋아질 것 같습니다